src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToFileScopedNamespaceAnalyzerTests.cs

```diff
@@ -349,7 +347,6 @@ class C
 ",
                 FixedCode = @"
 namespace $$N;
-
 // comment
```

Comment:
this feels like an improvement.  the original code effectively had the comment right underneath the namespace.  the new code preserves that.